msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md

```diff
@@ -30,6 +30,9 @@ Enable Teams for your organization and for more information, see [enabling Teams
 
 ## Enable custom Teams apps and turn on custom app uploading
 
+> [!IMPORTANT]
+> If you have installed an instant sandbox or a configurable sandbox with the Teams sample data pack, you can skip the steps to enable sideloading. The **Upload custom apps** setting is enabled by default in the Teams sample data pack.
```

Comment:
I suggest changing "If you have installed an instant sandbox" to "If you set up a Microsoft 365 Developer Program instant sandbox".

Reply:
updated as suggested.